### PR TITLE
Edits to reflect changes in CLI name and various functionality.

### PR DESCRIPTION
**Pull request by soniamitchell**
* Code runs should now me registered at the start of the run
* Write version number variables should be defined prior to the code run by the CLI

### CODE REVIEW

**Comment by soniamitchell** [comment]
More updates will be required after the registry is updated.

File: content/docs/interface/example1/_index.md [hunk 1]
```diff
@@ -225,9 +222,10 @@ finalise(handle)
 
 ### `finalise()`
 
-- rename the data product as *<hash>.h5*
-- record data product metadata (*e.g.* location, components, various descriptions, issues) in the data registry
-- record the code run in the data registry
+- renames any hdf5 files as *<hash>.h5*
+- renames any data products (storage directory) if variables are present, *e.g.* for `human/outbreak/simulation_run-{RUN_ID}`, `{RUN_ID}` is replaced with the CodeRun UUID
```
Comment (richardreeve) [inline content/docs/interface/example1/_index.md:226]: Won't `RUN_ID` have been determined in advance (at `initialise()`)?

**Review by richardreeve — COMMENTED** [review]
Looks good to me - just one query...

File: content/docs/interface/example1/_index.md (hunk 1 above)
Comment (soniamitchell) [reply content/docs/interface/example1/_index.md:226]: Yes, it's carried through to `finalise()` through the handle.

Comment (soniamitchell) [reply content/docs/interface/example1/_index.md:226]: See line 182 (I don't know how to quote code chunks here)

Comment (richardreeve) [reply content/docs/interface/example1/_index.md:226]: @soniamitchell As far as renaming any hdf5 files as *\<hash\>.h5* - I wonder whether that's still the right thing to do? Then you wouldn't be able to create the object in the registry until the very end, wouldn't you? Or can you still create it, but only make the storage location at the end?

Comment (soniamitchell) [reply content/docs/interface/example1/_index.md:226]: I wasn’t part of the naming files as a hash conversation so don’t know why it’s convention. 

The object and storage location entries are created in finalise (at the end?) anyway? so it doesn’t make any difference to me.

Comment (richardreeve) [reply content/docs/interface/example1/_index.md:226]: Okay, I'm happy then. Please do go ahead and merge. I may have a more serious go through the whole docs later in the week now I have a bit of time.

Comment (soniamitchell) [reply content/docs/interface/example1/_index.md:226]: Fab! I'll likely have to update again once the new schema is applied to the local registry so expect another PR soonish...

**Pull request merged**